server/utils.js

```diff
@@ -9,17 +10,22 @@ var getCrimeLocs = function (callback) {
 };
 
 var convertLatLngToStreet = function(lng, lat, callback) {
-  var url =  `http://locationiq.org/v1/reverse.php?format=json&key=6bc67043075bb2c0f0b3&lat=${lat}&lon=${lng}`;
+  var url =`https://maps.googleapis.com/maps/api/geocode/json?latlng=${lat},${lng}&key=${API_KEY}`;
   request(url, function(err, res, body) {
     if (err) {
       callback(err, null);
     } else {
-      var street = JSON.parse(body).address.road;
+      var street = JSON.parse(body).results[0].address_components[1].long_name;
       dbRating.findRatingEntry(street, function(err, results) {
-        if(err) {
+        if (err) {
           callback(err, null);
         } else {
-          callback(null, results);
+          if (results.length < 1) {
+            var defaultResponse = [{street: street, counter: 5, rating: 'red'}];
```

Comment:
Would it be helpful to change the default rating to something other than green/yellow/red for debugging purposes? or is it helpful that it is red?